agent/config/const_windows.go

```diff
@@ -15,5 +15,23 @@
 
 package config
 
-// OSType is the type of operating system where agent is running
+import "golang.org/x/sys/windows/registry"
+
 const OSType = "windows"
+const (
+	// envSkipDomainJoinCheck is an environment setting that can be used to skip
+	// domain join check validation. This is useful for integration and
+	// functional-tests but should not be set for any non-test use-case.
+	envSkipDomainJoinCheck  = "ZZZ_SKIP_DOMAIN_JOIN_CHECK_NOT_SUPPORTED_IN_PRODUCTION"
+	releaseId2004SAC        = "2004"
+	releaseId1909SAC        = "1909"
+	windowsServer2019       = "Windows Server 2019"
+	windowsServer2016       = "Windows Server 2016"
+	windowsServerDataCenter = "Windows Server Datacenter"
+	installationTypeCore    = "Server Core"
+	installationTypeFull    = "Server"
+	unsupportedWindowsOS    = "windows"
+	osTypeFormat            = "WINDOWS_SERVER_%s_%s"
```

Comment:
is there any reason why upper case should be used for instance capability?

Reply:
I believe you were asking about OSType, If that is correct the ECS Control Plane uses Upper case , so just used the same here for consistency

Reply:
could you add an example of what the OSType capability looks like for windows?